webpack.ssg.js

```diff
@@ -94,6 +94,41 @@ module.exports = (env) =>
           'loaders/url-loader': 'https://v4.webpack.js.org/loaders/url-loader',
           'loaders/file-loader':
             'https://v4.webpack.js.org/loaders/file-loader',
+          'configuration/other-options/#cache': 'configuration/cache',
```

Comment:
```suggestion
          'configuration/other-options/#cache': '/configuration/cache',
```
Need a prefix `/`. Otherwise it won't redirect, e.g., https://webpack-js-org-git-cache-page-webpack-docs.vercel.app/configuration/other-options/#cache will stay on other-options page.

Reply:
Still not working 😞

Reply:
I just noticed it too. Seems the [redirect plugin](https://github.com/bebraw/redirect-webpack-plugin/blob/master/index.js#L15) won't work against those fragments' redirection. I think we can do it on client side. For example, include another `useEffect(() => {})` in `Site.jsx` to achieve the redirections of fragment type.

```js
useEffect(() => {
// if the current pathname is /configuration/other-options/#cache
// redirect it to /configuration/cache
// same could be applied to others
}, []);
```

Reply:
Thanks, working now 🚀